.flake8

```diff
@@ -1,4 +1,4 @@
 [flake8]
 ignore = E203, E266, W503, E402
-max-line-length = 88
+max-line-length = 100
```

Comment:
This may cause many related formatting changes - try update `black`'s max-line-length config in `pyproject.yaml` to 100 and re-run `./dev/format.sh`, you will see a tones of formatting changes in the project, which we would like to avoid.

88 is the recommended default of `black`: https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length and we think it's better to just stick with that. If there are `max-line-length` linter error in the codebase that's very hard to fix, we can just ignore them in the PR and feel free to ping me in slack, I'd be happy to help find a solution.

Reply:
There are only 2 lines which are causing this error. If we can reformat those lines then it should be ok. Tried disabling but it's causing some issues when trying to put a comment inside another comment .Pinged you on slack regarding this.